castle/cms/tests/test_keywordmanager.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+from Products.PloneKeywordManager.tests.test_controlpanel import ControlPanelTestCase
+from Products.PloneKeywordManager.tests.test_dexterity import DexterityContentTestCase
+from Products.PloneKeywordManager.tests.test_non_ascii_keywords import NonAsciiKeywordsTestCase
+from Products.PloneKeywordManager.tests.test_setup import InstallTestCase
+from castle.cms.testing import CASTLE_PLONE_INTEGRATION_TESTING
+
+class ControlPanel(ControlPanelTestCase):
```

Comment:
What is going on with this file? Should we remove this if we're not going to have any tests related to it?

Reply:
The tests are inherited from the tests in Products.PloneKeywordManager